hack/windows/Prepare-Node.ps1

```diff
@@ -1,37 +1,30 @@
 <#
 .SYNOPSIS
-(Test Only)Assists with preparing a Windows VM prior to calling kubeadm join
+Assists with preparing a Windows VM prior to calling kubeadm join
 
 .DESCRIPTION
-This script is only used for test to assist with joining a Windows node to a cluster.
-For production environment please follow the antrea windows installation guide and use Kubernetes official PrepareNode.ps1
-script from the sig-windows-tools repo release page: https://github.com/kubernetes-sigs/sig-windows-tools/releases
```

Comment:
Is it supposed to be part of this PR? Have we determined to remove this disclaimer? @wenyingd @XinShuYang

Reply:
@NamanAg30 May I know why remove this part?

Reply:
Added this part back